quickwit/quickwit-control-plane/src/control_plane.rs

```diff
@@ -360,30 +454,21 @@ impl Handler<CreateIndexRequest> for ControlPlane {
     async fn handle(
         &mut self,
         request: CreateIndexRequest,
-        ctx: &ActorContext<Self>,
+        _ctx: &ActorContext<Self>,
     ) -> Result<Self::Reply, ActorExitStatus> {
-        let index_config = match metastore_serde_utils::from_json_str(&request.index_config_json) {
-            Ok(index_config) => index_config,
-            Err(error) => {
-                return Ok(Err(ControlPlaneError::from(error)));
-            }
-        };
-        let index_uid: IndexUid = match self.metastore.create_index(request).await {
-            Ok(response) => response.index_uid.into(),
+        let response = match self.metastore.create_index(request).await {
+            Ok(response) => response,
             Err(metastore_error) => return convert_metastore_error(metastore_error),
         };
-
-        let index_metadata: IndexMetadata =
-            IndexMetadata::new_with_index_uid(index_uid.clone(), index_config);
-
-        self.model.add_index(index_metadata);
-
-        self.rebuild_plan_debounced(ctx);
```

Comment:
ok so as there are no shards, we don't need that right? (I guess it's true only for ingest V2)

Reply:
Yeah, we shouldn't. The comment is confusing.